parsel/csstranslator.py

```diff
@@ -89,6 +101,116 @@ def xpath_text_simple_pseudo_element(self, xpath):
         """Support selecting text nodes using ::text pseudo-element"""
         return XPathExpr.from_xpath(xpath, textnode=True)
 
+    def xpath_first_of_pseudo(self, xpath):
```

Comment:
WIP mostly because of missing docs.
No idea about Travis tests failing with pypy.